packages/web-pkg/src/composables/keyboardActions/useKeyboardActions.ts

```diff
@@ -78,25 +78,38 @@ export const useKeyboardActions = (options?: KeyboardActionsOptions): KeyboardAc
       return
     }
 
+    type ModifierKey = 'altKey' | 'ctrlKey' | 'metaKey' | 'shiftKey'
+
     const { key, ctrlKey, metaKey, shiftKey } = event
     let modifier = null
+    const disallowedModifierKeys: ModifierKey[] = []
+
     if (metaKey || ctrlKey) {
-      modifier = ModifierKey.Ctrl
+      modifier = Modifier.Ctrl
+      disallowedModifierKeys.push('altKey', 'shiftKey')
     } else if (shiftKey) {
-      modifier = ModifierKey.Shift
+      modifier = Modifier.Shift
+      disallowedModifierKeys.push('altKey', 'ctrlKey', 'metaKey')
     }
 
+    const hasDisallowedModifier = (event: KeyboardEvent, disallowedModifierKeys: ModifierKey[]) =>
+      disallowedModifierKeys.some((key) => event[key])
+
     unref(actions)
       .filter((action) => {
-        return action.primary === key && action.modifier === modifier
+        return (
+          action.primary === key &&
+          action.modifier === modifier &&
+          !hasDisallowedModifier(event, disallowedModifierKeys)
```

Comment:
lol ... we don't need to check this in the loop, if we don't pass in the `action` anyway - we can just do

```typescript
if(disallowedModifierKeys.some((key) => event[key])) {
  return
}
```

above the loop.